docs/rings.md

```diff
@@ -0,0 +1,115 @@
+# Ring Managment 101
+
+## Ring Overview
+
+The hummingbird rings are responsible for mapping what devices data is stored on.  There is a separate ring for the objects, containers and accounts. Rings are created when you build a new cluster and updated when changes to the cluster need to be made like for example, adding or removing nodes or devices to the storage cluster.  It is important that all servers in the cluster have the same and most current version of the ring.
+
+### Zones
+
+A Zone is an abstract concept in the ring that allows you to define failure domains within the cluster.  Each copy of an object will be stored in a device in a different zone.  It is recommended to choose the highest level of isolation that makes sense for the cluster size.  For example for a small cluster, with a handful of machines, each server might be a zone.  For a larger cluster, it might be each rack of servers, etc.
+
+### Partitions
+
+When an object is stored in the cluster, it is not directly to a device, but instead to a partition via a hash.  The partitions are evenly distributed across all of the devices in the cluster.
```

Comment:
it is not directly assigned to/placed on(?) a device. Or is "stored" the verb here?